src/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './components/app';
+import rootReducer from './reducers/rootReducer'
+import { createStore, applyMiddleware, compose } from 'redux';
+import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
+
+const store = createStore(
+  rootReducer,
+  compose(
+    applyMiddleware(thunk),
+    window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__())
```

Comment:
```suggestion
    ...(window.__REDUX_DEVTOOLS_EXTENSION__ ? [window.__REDUX_DEVTOOLS_EXTENSION__()] : [])
```

@nicollis This is the cause of that issue you're seeing. Not having redux devtools installed causes this to fail and crash. (They just needed to not apply middleware in the event that devtools aren't installed.)

Reply:
Some inconsistent formatting in this...

- Lots of missing semicolons
- Inconsistent spacing (`if(` vs `if (`, `const {someImport} =` vs `const { someImport } =`)
- Closing parens alternate between same indentation and end of line (for longer statements)

Mostly minor style gripes easily fixed by Prettier / lint rules.